appinventor/components/src/edu/mit/appinventor/ble/BluetoothLE.java

```diff
@@ -71,13 +72,13 @@
         "The BluetoothLE extension requires Android 5.0 or higher to avoid known " +
         "issues with Google's Bluetooth LE support prior to Android 5.0.",
     category = ComponentCategory.EXTENSION,
-    versionName = "20240822",
+    versionName = "20241218",
```

Comment:
I will bump the version number when we do a release so you don't need to update it as part of the PR.